src/app/containers/ConsentBanner/useConsentBanners/index.test.js

```diff
@@ -0,0 +1,278 @@
+/* eslint-disable global-require */
+import { renderHook, act } from '@testing-library/react-hooks';
```

Comment:
This file used to be called `CanonicalLogic/index.test.js`. Check the diffs here:

https://www.diffchecker.com/ESqT1tOl